### PR TITLE
Feat/pn 386 fe pn ga visualizzazione delle notifiche ricevute persone giuridiche

### PR DESCRIPTION
**Pull request by maurizioflauti**
## Short description
Clone personagiuridica-webapp starting from personafisica-webapp with initial change requests

## List of changes proposed in this pull request
- Sidemenu changes
- Header changes

Here's the full list of all customized (starting from pf) files:
- buildspec.yaml
- CHANGELOG.md
- package.json
- sonarqube-scanner.js
- public/locales/it/common.json
- public/locales/it/notifiche.json
- src/App.tsx
- src/navigation/routes.const.ts
- src/navigation/routes.tsx
- src/pages/Contacts.page.tsx
- src/pages/Notifiche.page.tsx
- src/pages/UnderConstruction.page.tsx

## How to test
Run the application, login using any pf user and verify verify that notifications list, notification detail and contacts sections meet the requirements

### CODE REVIEW

File: packages/pn-personagiuridica-webapp/src/pages/UnderConstruction.page.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Box, Typography } from '@mui/material';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/UnderConstruction.page.tsx:2]: If this page is not used any more, remove it

File: packages/pn-personagiuridica-webapp/src/validators/CreateDelegationValidator.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Validator } from '@pagopa-pn/pn-validator';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/validators/CreateDelegationValidator.ts:1]: This file should not be used. delete it

File: packages/pn-personagiuridica-webapp/src/validators/PersonValidator.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Validator } from '@pagopa-pn/pn-validator';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/validators/PersonValidator.ts:1]: this file should not be used. delete it

File: packages/pn-personagiuridica-webapp/src/App.tsx [hunk 4]
```diff
@@ -44,85 +51,111 @@ const productsList: Array<ProductSwitchItem> = [
...
+    // init localization
+    initLocalization((namespace, path, data) => t(path, { ns: namespace, ...data }));
+    // eslint-disable-next-line functional/immutable-data
+    errorFactoryManager.factory = new PFAppErrorFactory((path, ns) => t(path, { ns }));
+  }, []);
+
+  useEffect(() => {
+    if (sessionToken !== '') {
+      void dispatch(getDomicileInfo());
+      void dispatch(getSidemenuInformation());
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/App.tsx:94]: This dispatch call Delegate api. Comment it out

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personagiuridica-webapp/public/index.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
   <meta http-equiv="x-ua-compatible" content="ie=edge" />
 
   <title>Piattaforma Notifiche</title>
-  <meta name="description" content="Piattaforma Notifiche per la persona giuridica" />
+  <meta name="description" content="Piattaforma Notifiche per il cittadino" />
```
[the remark this answers is not in the document]
Comment (maurizioflauti) [reply packages/pn-personagiuridica-webapp/public/index.html:12]: sure

File: packages/pn-personagiuridica-webapp/src/pages/UnderConstruction.page.tsx (hunk 1 above)
Comment (maurizioflauti) [reply packages/pn-personagiuridica-webapp/src/pages/UnderConstruction.page.tsx:2]: it's used when clicking on the sidebar items "Utenti" and "Gruppi"

File: packages/pn-personagiuridica-webapp/src/validators/PersonValidator.ts (hunk 3 above)
Comment (maurizioflauti) [reply packages/pn-personagiuridica-webapp/src/validators/PersonValidator.ts:1]: ok

File: packages/pn-personagiuridica-webapp/src/App.tsx (hunk 4 above)
Comment (maurizioflauti) [reply packages/pn-personagiuridica-webapp/src/App.tsx:94]: ok

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**